# sensorData.msg 

Header header

# Laser Scanners
float64 laser0_min_dist
float64 laser1_min_dist

# IMU's
float64 imu1_ang_x
float64 imu1_ang_y
float64 imu1_ang_z
float64 imu1_vel_x
float64 imu1_vel_y
float64 imu1_roll
float64 imu1_pitch
float64 imu1_yaw

float64 imu2_ang_x
float64 imu2_ang_y
float64 imu2_ang_z
float64 imu2_vel_x
float64 imu2_vel_y
float64 imu2_roll
float64 imu2_pitch
float64 imu2_yaw

float64 imu3_ang_x
float64 imu3_ang_y
float64 imu3_ang_z
float64 imu3_vel_x
float64 imu3_vel_y
float64 imu3_roll
float64 imu3_pitch
float64 imu3_yaw

float64 imu4_ang_x
float64 imu4_ang_y
float64 imu4_ang_z
float64 imu4_vel_x
float64 imu4_vel_y
float64 imu4_roll
float64 imu4_pitch
float64 imu4_yaw

float64 vz_1
float64 vz_2
float64 vz_3
float64 vz_4


float64 u1
float64 u2
float64 u3
float64 u4
